src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -40,9 +40,9 @@ public class ImportCommand extends Command {
     public static final String MESSAGE_NOT_IMPLEMENTED_YET =
             "Remark command not implemented yet";
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Imports contacts from specified filepath."
-            + " Must be an absolute filePath "
-            + "Parameters: filePath"
-            + "[" + PREFIX_IMPORT + "import]"
+            + " Must be an absolute filePath\n"
+            + "Parameters: filePath\n"
+            + "[" + PREFIX_IMPORT + "import]\n"
```

Comment:
looks good to me! it is important that the error message shown is clear so that users knows whats wrong